Failure to install plugin dependencies

Hello, I'm getting an error while trying to install the plugin's dependencies:
```
12:20:37 PM: ────────────────────────────────────────────────────────────────
12:20:37 PM:   Dependencies installation error                               
12:20:37 PM: ────────────────────────────────────────────────────────────────
12:20:37 PM: ​
12:20:37 PM:   Error message
12:20:37 PM:   Error while installing dependencies in /opt/build/repo/.netlify/plugins/
12:20:37 PM:   npm ERR! code E404
12:20:37 PM:   npm ERR! 404 Not Found - GET https://apigcp.nimbella.io/downloads/nim/nim-v1.8.0/nim-v1.8.0.tar.gz
12:20:37 PM:   npm ERR! 404
12:20:37 PM:   npm ERR! 404  'nimbella-cli@https://apigcp.nimbella.io/downloads/nim/nim-v1.8.0/nim-v1.8.0.tar.gz' is not in the npm registry.
12:20:37 PM:   npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
12:20:37 PM:   npm ERR! 404 It was specified as a dependency of 'netlify-plugin-nimbella'
12:20:37 PM:   npm ERR! 404
12:20:37 PM:   npm ERR! 404 Note that you can also install from a
12:20:37 PM:   npm ERR! 404 tarball, folder, http url, or git url.
```

https://app.netlify.com/sites/gatsby-starter-netlify-cms-erez/deploys/5fd0b29985acf600079497f8

It seems the following URL https://github.com/nimbella/netlify-plugin-nimbella/blob/a337de697c4ee8187ea84fa5f883bf6c86b7547e/package.json#L34 returns 404

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "make-dir": "^3.1.0",
     "netlify-lambda": "satyarohith/netlify-lambda#88eac6a37bfb0920897c86ecda8682944a2bf5be",
     "netlify-redirect-parser": "^2.5.0",
-    "nimbella-cli": "https://apigcp.nimbella.io/downloads/nim/nim-v1.8.0/nim-v1.8.0.tar.gz"
+    "nimbella-cli": "https://apigcp.nimbella.io/downloads/nim/nimbella-cli.tgz"
   },
   "xo": {
     "space": true,

